bin/util/bibblio_export

```diff
@@ -25,12 +26,24 @@ parser.add_argument(
     '--languages', type=unicode, nargs='*', default=u'eng',
     help='Look for Works in these languages'
 )
+parser.add_argument(
+    '--fiction', action='store_true',
+    help='Export fiction and nonfiction texts'
+)
```

Comment:
That's how it used to be, but I decided (at least for now) to keep fiction and nonfiction separate. Open to thoughts on this; I thought it could help recommendation results.